sec1_intro_and_setup/test_app/bin/setup

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby.exe
```

Comment:
Whats this file for?

Reply:
Allows one to write commands that will run when your app is first started.